flowey/flowey_lib_hvlite/src/build_openhcl_boot.rs

```diff
@@ -88,9 +81,12 @@ impl FlowNode for Node {
                 target,
                 no_split_dbg_info: false,
                 extra_env: Some(ReadVar::from_static(
-                    [("MINIMAL_RT_BUILD".to_string(), "1".to_string())]
-                        .into_iter()
-                        .collect(),
+                    [
+                        ("RUSTC_BOOTSTRAP".to_string(), "1".to_string()),
+                        ("MINIMAL_RT_BUILD".to_string(), "1".to_string()),
```

Comment:
I think if you're using the `minimal_rt` config files then this isn't necessary.